src/mobility/components/OperatorBenefit.tsx

```diff
@@ -37,14 +36,18 @@ export const OperatorBenefit = ({
       <Section>
         <GenericSectionItem>
           <View style={style.benefitContainer}>
-            <View style={style.benefitImage}>
               {isUserEligible && benefit.imageWhenActive && (
-                <SvgXml xml={benefit.imageWhenActive} />
+                <Image
```

Comment:
As discussed on Slack thread: Martin suggested that we shouldn't put the image (even as string) to firestore, as it is not the right way to store images, instead we should use URL on the firestore and put the assets somewhere else.

After Martin commented,  @gorandalum suggested that we should hold a talk with the designer to find the best solution, because right now, having operator png image on the indicator for active promotion is not scalable, might discuss on how to find the best way to implement this part.